From: Stephane Glondu <steph@glondu.net>
Date: Thu, 30 Jan 2020 13:06:08 +0100
Subject: read_main_debug_info: do not die in -custom executables

This fatal error occurred with some executables linked with -custom
(with the Debian-specific patch), notably ppx preprocessors during the
build of ppxlib.

---
 runtime/backtrace_byt.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/runtime/backtrace_byt.c b/runtime/backtrace_byt.c
index 8ea94ee..d82816c 100644
--- a/runtime/backtrace_byt.c
+++ b/runtime/backtrace_byt.c
@@ -342,7 +342,7 @@ static void read_main_debug_info(struct debug_info *di)
 
   fd = caml_attempt_open(&exec_name, &trail, 1);
   if (fd < 0){
-    caml_fatal_error ("executable program file not found");
+    /* caml_fatal_error ("executable program file not found"); */
     CAMLreturn0;
   }
 
